third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -68,27 +71,32 @@ func resourceBigtableInstance() *schema.Resource {
 							// so mark as computed.
 							Computed:     true,
 							ValidateFunc: validation.IntAtLeast(1),
+							Description: `The number of nodes in your Cloud Bigtable cluster. Required, with a minimum of 1 for a PRODUCTION instance. Must be left unset for a DEVELOPMENT instance.`,
 						},
 						"storage_type": {
 							Type:         schema.TypeString,
 							Optional:     true,
 							Default:      "SSD",
 							ValidateFunc: validation.StringInSlice([]string{"SSD", "HDD"}, false),
+							Description: `The storage type to use. One of "SSD" or "HDD". Defaults to "SSD".`,
 						},
 					},
 				},
+				Description: `A block of cluster configuration options. This can be specified 1 or 2 times. See structure below.`,
```

Comment:
Can you move this to where the other properties for `cluster` are defined? It's a bit hard to find all the way down here.

Reply:
Also, you can remove "See structure below"